scripts/ci/40_e2e.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -euxo pipefail
 
-yarn run e2e $1 --headless --production --record --group=$1
+yarn e2e-ci $1
```

Comment:
Would it work to pass and forward some of these arguments from this script, so users can more easily run `e2e-ci` on their machines?

Reply:
Yes, that would be ideal. `forwardAllArgs` was not working as I was hoping. But I will look closer at this, as I would prefer the user to be able to `yarn e2e-ci web-e2e` locally without needing to modify `workspace.json`